src/tilda/grammar/servlets/TildaSQLInit.java

```diff
@@ -39,22 +42,28 @@
 	  TildaSQLValidator _Validator;
 	  JsonObject json = new JsonObject();
 	  public void init() throws ServletException{}
-	  public void doGet(HttpServletRequest request,
+	  public void doPost(HttpServletRequest request,
 	                    HttpServletResponse response)
 	            throws ServletException, IOException
 	  {
 		  LogRequestHeader(21, request);
-		  PrintWriter _PrintWriter = response.getWriter();
-		  // Generate tabledef.
-		  for(ColumnDefinition colDef : ColDefs._COLS){
-			  // tableName: { columnName: "" }
-			  json.add(colDef.getName(), new JsonObject());
+		  String jsonStr = request.getParameter("payload");
+		  JsonElement jelement = new JsonParser().parse(jsonStr);
+		  JsonArray jArr = jelement.getAsJsonArray();
+		  for(JsonElement obj : jArr){
+			  JsonObject jsonObj       = (JsonObject)obj;
+			  String     _ColName      = jsonObj.get("name").getAsString();
+			  ColumnType _Type         = ColumnType.valueOf(jsonObj.get("type").getAsString());
+			  String     _Description  = jsonObj.get("desc").getAsString();
+			  boolean    _Multi        = Boolean.parseBoolean(jsonObj.get("name").getAsString());
+			  boolean    _Nullable     = Boolean.parseBoolean(jsonObj.get("nullable").getAsString());
+
+			  ColDefs._COLS.add(ColumnDefinition.Create(_ColName, _Type, _Multi, _Nullable, _Description));
```

Comment:
please check ColDefs.java. ColDef is a static collection instead it should be dependent on user's session, this collection will grow indefinitely and.there might be duplicates. I'm not sure whats the right approach.

@ldhasson